airflow/contrib/kubernetes/worker_configuration.py

```diff
@@ -213,8 +281,7 @@ def generate_dag_volume_mount_path(self):
     def make_pod(self, namespace, worker_uuid, pod_id, dag_id, task_id, execution_date,
                  try_number, airflow_command, kube_executor_config):
         volumes_dict, volume_mounts_dict = self.init_volumes_and_mounts()
```

Comment:
Can we rename this? The `init_` prefix on it is confusing me with `init_containers` - and I think now these volumes/mounts have nothing to do with the init_container, correct?

Reply:
My understanding is that it never was for the `init` container, it used to pass the return value to the init container function (see below), I think `init_volumes_and_mounts` was meant for "initialize volumes and mounts", I agree it's confusing but my code change doesn't make it different than it was

I'll rename for clarity

Reply:
Yup, your PR doesn't change it, but at least before it was also used by the init container, where as now it isn't :)

Thanks.